src/Email.php

```diff
@@ -11,12 +11,7 @@ public function render() {
     if (isset($link['query'])) {
       $query['query'] = $link['query'];
     }
-    $parts = explode('/', $query['path']);
-    if (count($parts) == 2 && $parts[0] == 'node' && is_numeric($parts[1])) {
-      $p_node = node_load($parts[1]);
-      $node = $p_node;
-    }
-    if ($node) {
+    if ($node = menu_get_object('node', 1, $query['path'])) {
```

Comment:
I'm pretty sure this breaks things as it will override the `$node` from the beginning of the function unconditionally. The old version only overrides `$node` when `$query['path']` points to a node path and leaves it untouched otherwise.